tests/pipeline/test_blaze.py

```diff
@@ -1876,6 +1876,54 @@ def test_checkpoints_out_of_bounds(self):
         self._test_checkpoints(checkpoints)
 
 
+class LastInGroupSortingTestCase(BlazeToPipelineTestCase):
```

Comment:
Yeah, but then other helper functions like`_test_id_macro` use `self.dates`. I was trying not to change the testing structure around too much.